common/src/main/java/com/microsoft/identity/common/internal/cache/AccountCredentialCache.java

```diff
@@ -12,23 +12,23 @@
 
 import java.util.List;
 
-public class MsalAccountCredentialCache
+public class AccountCredentialCache
         extends OAuth2TokenCache
         implements IShareSingleSignOnState {
 
     // SharedPreferences used to store Accounts and Credentials
     private final SharedPreferencesFileManager mAccountCredentialSharedPreferences;
 
     private List<IShareSingleSignOnState> mSharedSsoCaches;
-    private IMsalCacheConfiguration mCacheConfiguration;
+    private ICacheConfiguration mCacheConfiguration;
 
     // The names of the SharedPreferences file on disk.
     private static final String sAccountCredentialSharedPreferences =
             "com.microsoft.identity.client.account_credential_cache";
 
-    public MsalAccountCredentialCache(final Context context,
-                                      final List<IShareSingleSignOnState> sharedSsoCaches,
-                                      final IMsalCacheConfiguration cacheConfiguration) {
+    public AccountCredentialCache(final Context context,
```

Comment:
This class' name is going to change, as this layer is really going to be a pass-through for data on its way to a more encapsulated cache object which has no awareness of `strategy`, `request`, or `response` objects.